new/PRS14541PeterBrru.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14541PeterBrru" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Peter Brru</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-08-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">Peter Brru</persName>
```

Comment:
here https://archive.org/details/in.ernet.dli.2015.503297/page/n187/mode/2up  "Peter Bru" is mentioned as a translator who (with his son) accompanied the British mission of Wylde in 1883

Reply:
I tried to create a wikidata-lemma for him, but all I got was L1347920. I think, this is wrong. What shall I do to create a person with Q..... ?

https://www.wikidata.org/wiki/Lexeme:L1347920

Reply:
I cannot figure it out from this document, but I think, it is Wylde, Augustus Blandy (ca.1850-1909) (PRS10176Wylde) as one of the prominent members of the Wylde family: https://reed.dur.ac.uk/xtf/view?docId=ark/32150_s1j6731385f.xml

Reply:
wikidata creation https://www.wikidata.org/w/index.php?title=Special:NewItem&label=petros%20birru

As for Wylde, what I suggested was also to add the bibliographic reference to the source. Clearly it is Augustus Blandy Wylde, there is an entry about him in EAe and his name/initials are provided in the source quoted, it is easy to find the book references and the book cover

Augustus Blandy Wylde, '83 to '87 in the Soudan: With an Account of Sir William Hewett's Mission to King John of Abyssinia, I, London: Remington and Co, 1888, p. 180